src/froentend/pages/login/LoginForm.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./forms.css";
+
+function Loginform() {
+	return (
+		<div>
+			<main className="signup__container">
+				<div className="signup__div">
+					<form className="form">
+						<h2 className="form__name">Login</h2>
+						<div className="input__box">
+							<label>Email address </label>
+							<input id="Email" className="input" />
+							<p className="input__message">Wrong Email</p>
+						</div>
+						<div className="input__box">
+							<label>Password</label>
```

Comment:
`htmlFor` attribute can be used to bind input to corresponding label

Reply:
yes added htmlFor to labels